packages/perspective/index.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module '@jpmorganchase/perspective' {
+    /**** object types ****/
+    export enum TypeNames {
+        STRING = 'string',
+        FLOAT = 'float',
+        INTEGER = 'integer',
+        BOOLEAN = 'boolean',
+        DATE = 'date'
+    }
+
+    export type ValuesByType = {
+        [ key in TypeNames ]: Array<string>
```

Comment:
For primitives it's better to use string[] for arrays.

Reply:
https://palantir.github.io/tslint/rules/array-type/
we use "generic" on our project, hence Array<T> over T[]
but happy to change if you want me to

Reply:
Sure, since there's no tslint setup of similar and they're equivalent let's just go with this for now. We can always change later.